.travis.yml

```diff
@@ -20,6 +20,10 @@ matrix:
       compiler: clang
       env: [ ASAN=ON ]
 
+    - os: linux
```

Comment:
Great to have one buildjob more. Is it possible to have a short description of the build jobs? (what means FULL)

Reply:
`FULL=ON` enables a “full build“. It is basically a shorthand for `BUILD_FULL=ON`.